src/main/java/keycontacts/model/student/GradeLevel.java

```diff
@@ -0,0 +1,119 @@
+package keycontacts.model.student;
+
+import java.util.regex.PatternSyntaxException;
+
+/**
+ * Represents a Student's grade level field in the student object.
+ */
+public class GradeLevel {
+
+    public static final String MESSAGE_CONSTRAINTS = "Grade level should only contain alphanumeric characters "
+            + "and spaces, exam board (e.g. ABRSM, Trinity) "
+            + "and grade level should be separated with a space. Grade level should not exceed 50 characters, "
+            + "and grade should be an integer between 1 and 12";
+
+    public final String value;
+
+    private String schoolLevel;
+    private int grade;
+
```

Comment:
I think there is no need for these fields, as I don't foresee us needing them. But KIV for now, we can clean it up in v 1.3

Reply:
But in case we want to implement something like increase grade level in the future (or at least just for flexibility purpose) I don't think we need to remove them

Reply:
the reason is that its increasing code complexity right now. I think it would be better if we just enforced that gradelevel followed a regex of '[Exam board] [number]' (see other model classes for the validation format). If needed in the future, then the granular information can be retrieved with a new method. I think it is cleaner than having extra attributes